### ISSUE TITLE
pickup problem

### ISSUE THREAD

**Issue by Cronosus** [BlackAce21/FakePlayerFix#3]
we just hit small problem on 1.7.10 server with robot from OpenComputers, 
for some reason they are unable to pick up dropped blocks/items when they mine.
even when they have all allowed via this plugin,

i'm little surprised we didnt hit it much much sooner. :D

**Comment by BlackAce21** [comment]
I just saw this as I am not as active lately due to irl stuff. I can add event hooks for item pickup events if you still need. Someone else was asking about wild card functions as well for the names cause some mods add player names to the fake player name based on the owner. I'll make both these adjustments tonight or tomorrow.

**Comment by Cronosus** [comment]
yeah, that would be nice too. Opencomputers do that naming this way now too, but old way still works. i just have problem with this pickup.

**Comment by BlackAce21** [comment]
Just a tiny update. I've actually decided to just rewrite the whole thing and make it smarter. This solution was created in an afternoon many years ago now and the code is so basic I can't make it do what I want without changing it completely. It's gonna take me a little bit as I'm so busy outside of this but it will get done :) keep eyes out.

**Comment by Cronosus** [comment]
ok, life is more important
